civihr_employee_portal/tests/Webform/LocationTypeIDConvertorTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Drupal\civihr_employee_portal\Webform\LocationTypeIDConvertor;
+
+class LocationTypeIDConvertorTest extends PHPUnit_Framework_TestCase {
+
+  /**
+   * @var array
+   */
+  protected $locationType;
+
+  protected function setUp() {
+    $result = civicrm_api3('LocationType', 'create', ['name' => 'Test']);
+    $this->locationType = array_shift($result['values']);
+  }
+
+  protected function tearDown() {
+    civicrm_api3('LocationType', 'delete', ['id' => $this->locationType['id']]);
+  }
+
+  public function testEmptyMapping() {
```

Comment:
This should probably be something like `testLocationTypeMappingIsNotEmpty` or something.
The current test name gives the impression that we would be asserting that that the mapping should be empty.